metricbeat/module/elasticsearch/enrich/_meta/fields.yml

```diff
@@ -15,11 +15,11 @@
           type: long
           description: >
             Current number of outstanding remote requests.
-        - name: total
+        - name: count
```

Comment:
@ycombinator I have changed this according to https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html from `total` to `count`. Do you think that I should also make an alias to maintain a mapping to the old field path?

Reply:
I would suggest not making this change for now, as it's a breaking change. Let's do any breaking changes in follow up PRs that we would merge into `master` (for 8.0.0) but not backport.

Reply:
Sounds good! :) Reverted! Thanks!